### PR TITLE
Updated package.json and CHANGELOG for release 4.0.1

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.1](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 4.0.1 (20201-01-15)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md:1]: `20201` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+- Changed the `SkyDataManagerColumnPickerSortStrategy` export to be referencable. [#22](https://github.com/blackbaud/skyux-data-manager/pull/22)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:2]:
```suggestion
- Updated the `SkyDataManagerColumnPickerSortStrategy` export to be referenceable. [#22](https://github.com/blackbaud/skyux-data-manager/pull/22)
```

File: CHANGELOG.md (hunk 1 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md:1]:
```suggestion
# 4.0.1 (2021-01-15)
```

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.1](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.1)

**Pull request merged**